database/migrations/2022_06_05_223148_add_parent_id_to_comments_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('comments', function (Blueprint $table) {
+            $table->unsignedBigInteger('parent_id')->after('id')->nullable();
+            $table->foreign('parent_id')->references('id')->on('comments');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('comments', function (Blueprint $table) {
+            $table->dropColumn('parent_id');
```

Comment:
If you are keeping the `down` (I would maybe remove it) you also need to drop the foreign key first right?

Reply:
Yes, I will grab that along with the next update I'm pushing.